File: Estrutura_Repetição/C++/Ex1113/Main.cpp
#include <iostream>

int main () {

    while (1) {
        int x, y;
        std::cin >> x >> y;

        if (x > y) {
            std::cout << "Decrescente" << std::endl;
        } else if (x < y) {
            std::cout << "Crescente" << std::endl;
        } else if (x == y) {
            break;
        }
    }

    return 0;
}

File: Estrutura_Repetição/C#/Ex1113/Main.cs
using System;

class MainClass {
    static void Main(string[] args) {
        int x, y;

        while (true) {
            string[] input = Console.ReadLine().Split();
            x = int.Parse(input[0]);
            y = int.Parse(input[1]);

            if (x > y) {
                Console.WriteLine("Decrescente");
            } else if (x < y) {
                Console.WriteLine("Crescente");
            } else if (x == y) {
                break;
            }
        }
    }
}